src/librustc_lexer/src/lib.rs

```diff
@@ -102,10 +181,9 @@ pub fn tokenize(mut input: &str) -> impl Iterator<Item = Token> + '_ {
     })
 }
 
-// See [UAX #31](http://unicode.org/reports/tr31) for definitions of these
-// classes.
-
 /// True if `c` is considered a whitespace according to Rust language definition.
+/// See [UAX #31](http://unicode.org/reports/tr31) for definitions of these
+/// classes.
```

Comment:
Nitpick, but I'd rather avoid referencing UAX 31 from public docs, and reference the Rust reference instead. This functions parses whitepsace according the definition of the rust langauge, and using UAX 31 is an implementation detail of that definition.